crates/proof-of-sql-planner/src/util.rs

```diff
@@ -29,6 +78,38 @@ pub(crate) fn scalar_value_to_literal_value(value: ScalarValue) -> PlannerResult
         ScalarValue::Int64(Some(v)) => Ok(LiteralValue::BigInt(v)),
         ScalarValue::UInt8(Some(v)) => Ok(LiteralValue::Uint8(v)),
         ScalarValue::Utf8(Some(v)) => Ok(LiteralValue::VarChar(v)),
+        ScalarValue::TimestampSecond(Some(v), Some(tz)) => {
+            let timezone = arrow_timezone_to_posql_timezone(&tz)?;
+            Ok(LiteralValue::TimeStampTZ(
+                PoSQLTimeUnit::Second,
+                timezone,
+                v,
+            ))
```

Comment:
I don't know of any use-cases for timezones at this point, and I'm ok with removing functionality in this specific case since the code would be drastically simpler.
```suggestion
        ScalarValue::TimestampSecond(Some(v), None) => Ok(LiteralValue::TimeStampTZ(
            PoSQLTimeUnit::Second,
            PoSQLTimeZone::utc(),
            v,
        )),
```